### ISSUE TITLE
It's too easy to delete a configuration by clicking on the "x" button on the tab

### ISSUE THREAD

**Issue by omajid** [philipphoffmann/gnome3-jenkins-indicator#74]
The settings dialog shows a tab for each jenkins instance. It's too easy to delete the configuration by accidentally clicking on the "x" which appears next to the tab name.  I accidentally clicked on it today "x" when I really just wanted to open up the settings for that instance.

Since the operation is destructive and there's no undo, perhaps a confirmation prompt can be added?

**Comment by sergiy-kozak** [comment]
Hi,
same happened to me once, would be indeed better to have the confirmation popping up.

**Comment by philipphoffmann** [comment]
yeah, I totally agree, feel free to provide a pull request, I would be happy to merge
